### DISCUSSION TITLE
[Projects Beta Bug?] Manually Selecting Sprint Timeframe Not Working?

### DISCUSSION THREAD

**Discussion by apintols** [community/community#7546] [Projects and Issues]
It seems that the option to change the date range for an iteration is not working or is not enabled.
<img width="1107" alt="GH Projects Beta Sprints" src="https://user-images.githubusercontent.com/69682363/141142310-76f4ccbf-104a-4f85-b597-dd09da6a3c32.png">

Attempting to click the date changes the background of the date range as if it were responding like a button, but it doesn't actually open anything up or allow the date to change.

Not sure if this is unfinished or bugged, but it's definitely a feature that would be useful!

**Comment by aznn** [comment]
We do 2 sprints a month, from 1st to 15th, and 15th to end of month. The current 2 week option does not align exactly with our sprint dates. 

Ability to select the sprint start date will be very helpful!

**Comment by skullface** [comment]
I experienced an error today when editing the first iteration start date to be a day in the future~ (Chrome 96, MacOS 11.6)

https://user-images.githubusercontent.com/221550/143278467-9211b004-6343-40b4-b276-608e3ca78ced.mp4

**Comment by skullface** [comment]
Update: This works as expected now 💕!
